OutOfSchool/OutOfSchool.DataAccess/Repository/IRatingRepository.cs

```diff
@@ -8,7 +8,4 @@ namespace OutOfSchool.Services.Repository;
 
 public interface IRatingRepository : IEntityRepository<long, Rating>
```

Comment:
same as above, use `IEntityRepository<long, Rating>`, you don't need custom interface unless it has methods

Reply:
The interface and the repository were removed.